src/components/skinManager.js

```diff
@@ -1,154 +1,9 @@
 define(['apphost', 'userSettings', 'browser', 'events', 'pluginManager', 'backdrop', 'globalize', 'require', 'appSettings'], function (appHost, userSettings, browser, events, pluginManager, backdrop, globalize, require, appSettings) {
     'use strict';
 
-    var currentSkin;
```

Comment:
Because this was doing nothing. Theme support is visible in the clients, skin support is not supported. When i was working out this code, i asked in riot for this functionality, and they said this never worked. Can't remember who said it, but i confirmed it and removed the associated code accordingly.